src/App.css

```diff
@@ -29,11 +29,16 @@
 }
 
 .draggable-element,
-.div-element {
+.div-element {  
```

Comment:
Let's remove the unnecessary whitespace.

Reply:
https://github.com/Yosabba/page-builder/pull/1/commits/db7063dd580d61d6c473695c543f73e2ef17d8f4